django/getting-started/existing.html.md

```diff
@@ -59,29 +68,36 @@ Machine 217811c53e0896 is created
 Postgres cluster django-existing-app-db created
   Username:    postgres
   Password:    lt5JoEIVons5INJ
-  Hostname:    django-existing-app-db.internal
+  Hostname:    django-existing-app-db.flycast
+  Flycast:     fdaa:3:9f45:0:1::2
   Proxy port:  5432
   Postgres port:  5433
-  Connection string: postgres://postgres:lt5JoEIVons5INJ@django-existing-app-db.internal:5432
+  Connection string: postgres://postgres:lt5JoEIVons5INJ@django-existing-app-db.flycast:5432
 
 Save your credentials in a secure place -- you won't be able to see them again!
 
 Connect to postgres
-Any app within the Jane Smith organization can connect to this Postgres using the following credentials:
-For example: postgres://postgres:lt5JoEIVons5INJ@django-existing-app-db.internal:5432
+Any app within the Jane Smith organization can connect to this Postgres using the above connection string
 
-
-Now that you've set up postgres, here's what you need to understand: https://fly.io/docs/reference/postgres-whats-next/
+Now that you've set up Postgres, here's what you need to understand: https://fly.io/docs/postgres/getting-started/what-you-should-know/
+Checking for existing attachments
+Registering attachment
+Creating database
+Creating user
 
 Postgres cluster django-existing-app-db is now attached to django-existing-app
 The following secret was added to django-existing-app:
-  DATABASE_URL=postgres://django_existing_app:6YuIWrdwzVOgVe1@top2.nearest.of.django-existing-app-db.internal:5432/django_existing_app?sslmode=disable
+  DATABASE_URL=postgres://django_existing_app:lt5JoEIVons5INJ@django-existing-app-db.flycast:5432/django_existing_app?sslmode=disable
 Postgres cluster django-existing-app-db is now attached to django-existing-app
-? Would you like to set up an Upstash Redis database now? No
+Wrote config file fly.toml
+
+[INFO] Python 3.10.12 was detected. 'python:3.10-slim-bullseye' image will be set in the Dockerfile.
 
-Your Django app is almost ready to deploy!
+Validating ~/django-existing-app/fly.toml
+Platform: machines
+✓ Configuration is valid
 
-We recommend using the database_url(pip install dj-database-url) to parse the DATABASE_URL from os.environ['DATABASE_URL']
```

Comment:
I think it's not necessary since we mentioned at the beginning: 

> The database adapter [Psycopg](https://www.psycopg.org/psycopg3/docs/) is commonly used along with [dj-database-url](https://pypi.org/project/dj-database-url/) to establish a Django database connection via a DATABASE_URL environment variable.